environments/backfillStorage.toml

```diff
@@ -3,190 +3,10 @@
     name     = "transformerExporter"
     save     = true
     schema   = "maker"
-    transformerNames = ["clip_aave_a_v1_6_0", "clip_bal_a_v1_6_0", "clip_bat_a_v1_6_0", "clip_comp_a_v1_6_0", "clip_eth_a_v1_5_0", "clip_eth_b_v1_5_0", "clip_eth_c_v1_5_0", "clip_guniv3daiusdc_1_v1_9_6", "clip_guniv3daiusdc_2_v1_9_12", "clip_knc_a_v1_6_0", "clip_link_a_v1_3_0", "clip_lrc_a_v1_6_0", "clip_mana_a_v1_6_0", "clip_matic_a_v1_9_4", "clip_renbtc_a_v1_6_0", "clip_uni_a_v1_6_0", "clip_univ2aaveeth_a_v1_8_0", "clip_univ2daieth_a_v1_8_0", "clip_univ2daiusdt_a_v1_8_0", "clip_univ2ethusdt_a_v1_8_0", "clip_univ2linketh_a_v1_8_0", "clip_univ2unieth_a_v1_8_0", "clip_univ2usdceth_a_v1_8_0", "clip_univ2wbtcdai_a_v1_8_0", "clip_univ2wbtceth_a_v1_8_0", "clip_wbtc_a_v1_5_0", "clip_wbtc_b_v1_9_10", "clip_wbtc_c_v1_9_11", "clip_wsteth_a_v1_9_8", "clip_yfi_a_v1_5_0", "clip_zrx_a_v1_6_0"]
```

Comment:
@jeannettemcd any idea why this all got removed/replaced with just `vat`?

Reply:
This is a backfill for vat_lite.